eng/pipelines/docs.yml

```diff
@@ -1,3 +1,6 @@
+#Pipeline variables:
+# $(additionalArgs) eg : -i "exc" --exc arm-advisor arm-apimanagement OR  -i "inc" --exc arm-advisor arm-apimanagement
```

Comment:
Yes! So we can now parameterize this thing entirely!

Reply:
Yes :)